api/src/main/java/uk/gov/hmcts/reform/divorce/caseapi/controllers/SaveAndCloseController.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.divorce.caseapi.controllers;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.divorce.caseapi.model.CcdCallbackRequest;
+import uk.gov.hmcts.reform.divorce.caseapi.notification.handler.SaveAndSignOutNotificationHandler;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@Slf4j
+@RestController
+public class SaveAndCloseController {
+
+    @Autowired
+    private SaveAndSignOutNotificationHandler saveAndSignOutNotificationHandler;
+
+    @PostMapping(path = "/notify-applicant", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
```

Comment:
@PostMapping(path = SUBMITTED_WEBHOOK)

I want the convention to be `/${event-name}/${callback-type}`